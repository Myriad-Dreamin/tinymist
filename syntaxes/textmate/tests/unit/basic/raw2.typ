
````typ
#(let a
````

== A

- a
- b

```typ
#(let a
```
