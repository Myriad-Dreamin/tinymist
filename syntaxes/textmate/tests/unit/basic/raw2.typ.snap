>
>````typ
#^^^^ source.typst markup.raw.block.typst punctuation.definition.raw.begin.typst
#    ^^^ source.typst markup.raw.block.typst fenced_code.block.language.typst
>#(let a
#^ source.typst markup.raw.block.typst meta.embedded.block.typst keyword.control.hash.typst
# ^ source.typst markup.raw.block.typst meta.embedded.block.typst meta.brace.round.typst
#  ^^^ source.typst markup.raw.block.typst meta.embedded.block.typst meta.expr.let.typst storage.type.typst
#     ^ source.typst markup.raw.block.typst meta.embedded.block.typst meta.expr.let.typst
#      ^ source.typst markup.raw.block.typst meta.embedded.block.typst meta.expr.let.typst variable.other.readwrite.typst
>````
#^^^^ source.typst markup.raw.block.typst punctuation.definition.raw.end.typst
>
>== A
#^^ source.typst markup.heading.typst punctuation.definition.heading.typst
#  ^ source.typst markup.heading.typst
#   ^ source.typst markup.heading.typst
>
>- a
#^^ source.typst punctuation.definition.list.unnumbered.typst
#  ^^ source.typst
>- b
#^^ source.typst punctuation.definition.list.unnumbered.typst
#  ^^ source.typst
>
>```typ
#^^^ source.typst markup.raw.block.typst punctuation.definition.raw.begin.typst
#   ^^^ source.typst markup.raw.block.typst fenced_code.block.language.typst
>#(let a
#^ source.typst markup.raw.block.typst meta.embedded.block.typst keyword.control.hash.typst
# ^ source.typst markup.raw.block.typst meta.embedded.block.typst meta.brace.round.typst
#  ^^^ source.typst markup.raw.block.typst meta.embedded.block.typst meta.expr.let.typst storage.type.typst
#     ^ source.typst markup.raw.block.typst meta.embedded.block.typst meta.expr.let.typst
#      ^ source.typst markup.raw.block.typst meta.embedded.block.typst meta.expr.let.typst variable.other.readwrite.typst
>```
#^^^ source.typst markup.raw.block.typst punctuation.definition.raw.end.typst
>